### PR TITLE
SB-1360 storage Id and repository Id naming

### PR DESCRIPTION
**Pull request by fuss86**

### CODE REVIEW

File: docs/user-guide/ui/storage-management.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Intro
```
Comment (carlspring) [inline docs/user-guide/ui/storage-management.md:2]: Typo: Module

Comment (fuss86) [reply docs/user-guide/ui/storage-management.md:2]: fixed :)

**Review by steve-todorov — APPROVED** [review]
LGMT :)

**Comment by carlspring** [comment]
@fuss86 : Could you please resolve the merge conflicts? Thanks! :)

LGTM and it's otherwise ready for merging!

**Pull request merged**